File: atcoder/abc154C_distinct_or_not.cpp
// https://atcoder.jp/contests/abc154/tasks/abc154_c
#include<iostream>
#include<set>

using namespace std;

int main() {
    int n;
    cin >> n;

    set<int> values;
    int value;
    for (int i = 0; i < n; i++) {
        cin >> value;
        values.insert(value);
    }

    if (int(values.size()) == n) cout << "YES" << endl;
    else cout << "NO" << endl;
    return 0;
}

File: atcoder/abc154C_distinct_or_not_.py
# vicfred
# https://atcoder.jp/contests/abc154/tasks/abc154_c
# sorting
n = int(input())
a = list(map(int, input().split()))

a.sort()

repeated = False
for i in range(len(a)-1):
    if a[i] == a[i+1]:
        repeated = True
        break

if repeated:
    print("NO")
else:
    print("YES")

